1-Draft/RFC00XX-Update-Error-View.md

```diff
@@ -0,0 +1,181 @@
+
+PS C:\test> Resolve-ErrorRecord
+
+PSMessageDetails      :
+Exception             : System.Management.Automation.ItemNotFoundException: Cannot find path 'C:\blah' because it does
+                        not exist.
+                           at System.Management.Automation.LocationGlobber.ExpandMshGlobPath(String path, Boolean
+                        allowNonexistingPaths, PSDriveInfo drive, ContainerCmdletProvider 
+
+```
+
+__Example 2__
+
+```powershell
+PS C:\test> slkjdfh
+ERROR: The term 'slkjdfh' is not recognized as the name of a cmdlet, function, script file, or operable program. At line:1 char:1
+PS C:\test> $error[0] | Resolve-ErrorRecord
+
+PSMessageDetails      :
+Exception             : System.Management.Automation.CommandNotFoundException: The term 'slkjdfh' is not recognized as
+```
+
+## Alternate Proposals and Considerations
+
+__Alternative single-line display__
+
+```powershell
+<CommandName>:<Exception Message>
+get-item: Cannot find path ‘C:\blah’ because it does not exist:
+
+<CommandName>:<Exception Message>: <Position>
+get-item: Cannot find path ‘C:\blah’ because it does not exist: At line:1 char:1
+
+<CommandName>:<Alias>:<Exception Message>: <Position>
+get-item: gi: Cannot find path ‘C:\blah’ because it does not exist: At line:1 char:1
+
+ERROR:<Exception Message>
+ERROR: Cannot find path ‘C:\blah’ because it does not exist
+
+ERROR:<Exception Message>: <Position>
+ERROR: Cannot find path ‘C:\blah’ because it does not exist: At line:1 char:1
+
+ERROR:<CommandName>:<Alias>:<Exception Message>: <Position>
+ERROR: get-item: gi: Cannot find path ‘C:\blah’ because it does not exist: At line:1 char:1
+```
+
+__Alternative color for all errors__
+
+1. We could change the default error message color from the current RED foreground and BLACK background to...
+2. Differentiating errors based on termination condition: terminating versus non-terminating is currently not intuitive. We are examining differentiating these conditions on the console. Example, adding a new property $host.PrivateData.NonTerminatingErrorForegroundColor ='Red'.
+For occasional customers, all error messages remain as color Red. For advanced customers, they can change non-terminating errors to another color to separate the error termination type in the console.
+
+__Alternative For Error, Warning, Verbose__
+
+1. We could be more terse in our messages and increase consistency with verbose and warning messages by using a single letter to identify the message.
+
+Legend: V = Verbose, W = Warning, E = Error(non-terminating future), F = Fatal
+
+```powershell
+V: You are running your code - what could possibly go wrong.
+
+W: You are about to try something that probably will not work.
+
+E: Your something broke, but Im still running. At line:1 char:1
+
+E: Your something broke, but Im still running. At line:1 char:1
+
+E: Your something broke, but Im still running. At line:1 char:1
+
+F: Now you really broke me. At line:1 char:1
+
+```
```

Comment:
Please, not this. Single letters are not searchable.

Reply:
@KirkMunro It could always be another view. SuperConsiseView :)

Reply:
Hey @KirkMunro -- Thanks! I do like the terseness. Error, Warning, Verbose seems like it could be abbreviated, but I get your point.